### PR TITLE
Fix #73. Change sprintf to json_encode in content data

### LINKED ISSUE #73: Escape double quotes when create TXT domain record with double quotes
The problem is in file DigitalOceanV2\Api\DomainRecord line 71
If i have a TXT record with double quotes, eg
"v=spf1 mx include:mailserver ?all"
the result is
{"name":"mydomain", "type":"TXT", "data":""v=spf1 mx include:mailserver ?all""}
Causing error 500

### PR DESCRIPTION
**Pull request by fabiopaiva**

### CODE REVIEW

File: src/Api/Domain.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ public function getByName($domainName)
     public function create($name, $ipAddress)
     {
         $headers = array('Content-Type: application/json');
-        $content = sprintf('{"name":"%s", "ip_address":"%s"}', $name, $ipAddress);
+        $content = json_encode(array('name' => $name, 'ip_address' => $ipAddress));
```
Comment (toin0u) [inline src/Api/Domain.php]: As this package uses php `>=5.4.0`, you can use the short array format :)

**Comment by toin0u** [comment]
Can you look at the specs as well ? :)
@yassirh What do you think ?

**Comment by yassirh** [comment]
@toin0u i'm fixing the specs... and it looks good.

**Comment by toin0u** [comment]
:+1:

**Comment by toin0u** [comment]
@fabaopaiva thank you :)

**Pull request merged**